tendrl/commons/utils/etcd_utils.py

```diff
@@ -0,0 +1,68 @@
+import etcd
+import __builtin__
+
+'''
+   Read from etcd
+   input params:
+       key  : type  - >  string
+              value - >  attr to be fetched
+
+   return param:
+       dict - > if read is successful
+       None - > if key is not found
+'''
+ 
+def read(key):
+    try:
+        return NS._int.client.read(key)
+    except (etcd.EtcdConnectionFailed, etcd.EtcdException) as ex:
```

Comment:
I have seen cases where when reads and/or writes by multiple tendrl processes failed (almost) parallelly, each of them try a reconnect. And what happens in reconnect is the client is first reset to null and then set to a new etcd client.. So what I feel is the best and/or easy approach is,  to rather gracefully accept AttributeError(and raise it off course so that the users of the utility can custom handle it) As this is expected to happen consecutive ly only a fixed number of times (equal to number of tendrl processes parallelly making those reads and/or writes) and this I feel will not land us into any sort of infinite maze :-) or not even will it cause any race condition...